GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -256,23 +256,23 @@ private void InitializeComponent()
             this.stageFileToolStripMenuItem.Image = global::GitUI.Properties.Images.Stage;
             this.stageFileToolStripMenuItem.Name = "stageFileToolStripMenuItem";
             this.stageFileToolStripMenuItem.Size = new System.Drawing.Size(296, 22);
-            this.stageFileToolStripMenuItem.Text = "Stage file(s)";
+            this.stageFileToolStripMenuItem.Text = "Sta&ge file(s)";
```

Comment:
I suggest that this is set as `s` as the hotkey, even if there is a conflict with `Sort`, I believe that is handled
(I wrote a comment about this but it was not committed it seems)

Reply:
> ... conflict with `Sort`, I believe that is handled

Yes, it *is* handled. But it needs at least one - unexpected - press of `Enter`. That's why I prefer unique mnemonics over "speaking" ones - since they can be seen all the time, and we have a perfect hotkey.
If you insist, I will not deny to change it because I will not use this particular mnemonic.

Reply:
Not insisting, but it seem disparate to have a separate hotkey and mnemonics.
It is only a conflict in Worktree

Reply:
done, just for once